code/src/Summary.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export const Summary = ({ consumption, numberOfBooks, favouriteAuthor, onRestart}) => {
+    return (
+        <>
+            <section className="summary">
+                <p>Wow! You read {numberOfBooks} books each year!</p>
+                <p>It's awesome to {consumption}.</p>
+                <p>{favouriteAuthor} is a good choice.</p>
+            </section>
+            <section className="summary white">
```

Comment:
same with the className here, i think it's missing a - and should be " summary-white" :)